core/src/main/java/tech/pegasys/web3signer/core/Eth1Runner.java

```diff
@@ -89,6 +89,7 @@ private ArtifactSignerProvider loadSigners(final Config config, final Vertx vert
             hashicorpConnectionFactory,
             config.getKeyConfigPath(),
             azureFactory,
+            vertx,
```

Comment:
this should probably be abstracted as a yubikeyHsmConnectionFactory, rather than passing down the raw vertx - similar t how we handle Hashicorp.